src/ext_manifest.c

```diff
@@ -24,10 +25,14 @@ const struct ext_man_header ext_man_template = {
 
 static int ext_man_open_file(struct image *image)
 {
-	/* open extended manifest outfile for writing */
-	sprintf(image->out_ext_man_file, "%s.xman", image->out_file);
-	unlink(image->out_ext_man_file);
```

Comment:
Dont we need to delete the old file ?

Reply:
Old file is deleted by `create_file_name` function.